docs: add 4.6.c release notes DOC-1868 (#6760)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

#### Breaking Changes {#breaking-changes-4.6.c}

:::warning

Due to updates to Cluster API (CAPI) in this release, we recommend that you enable the
[Pause Agent Upgrades](../clusters/cluster-management/platform-settings/pause-platform-upgrades.md) feature on any

> impacted clusters until you are ready for the cluster repaves.

Reviewer: Can we slightly tweak this to ".... on any impacted clusters until you've handled the below mentioned breaking changes and are ready for cluster repaves."?

Author: Added this!

:::

- Due to new behavior introduced with Cluster API (CAPI) v1.9.4, you must add the `cluster.x-k8s.io/drain: skip` label

>   to any deployments with the `Node.spec.unschedulable` field set. If not added, this can lead to deployments stuck in a

Reviewer: with the `Node.spec.unschedulable` ~field~ toleration set.

Author: Added this!

  Refer to the [Create and Manage Azure IaaS Cluster](../clusters/public-cloud/azure/create-azure-cluster.md) guide for
  more information.

> - Once you install Palette or upgrade to version 4.6.c or later, you have 30 days to activate it. During this time, you
>   have unrestricted access to all of Palette's features. After 30 days, you can continue to use Palette, and existing
>   clusters will continue to run, but you cannot perform the following day-2 operations until Palette is activated.
> 
>   Refer to the Activate Palette guide for further information.

Reviewer 2: ```suggestion
- Beginning with version 4.6.c, when you install [self-hosted Palette](../enterprise-version/enterprise-version.md) or [Palette VerteX](../vertex/vertex.md) or upgrade an existing installation to version 4.6.c or later, you enter a 30-day trial period. During this time, you have access to all features. After 30 days, functionality is restricted; deployed clusters will continue to run, but you cannot perform [day-2 operations](../clusters/cluster-management/cluster-management.md) or deploy additional clusters until your installation is activated. Activation keys are included with your subscription and are no additional cost.

  Refer to the activation guides for [Palette] and [Palette VerteX] to learn how to activate your installation.
```

Links for later substitution here: 
- Palette - ../enterprise-version/activate-installation/activate-installation.md
- Palette VerteX - ../vertex/activate-installation/activate-installation.md

  :::

- The Palette UI allows users to specify an optional team description during team creation. This new field makes it
  possible for system administrators to provide a meaningful description to user teams. Refer to the
  [Create and Manage a Team](../user-management/users-and-teams/create-a-team.md) guide for further information.

> - Palette [interface customization](../enterprise-version/system-management/customize-interface.md) has been expanded to
>   include sidebar customization, documentation links, and all login screen logos.

Reviewer 2: ```suggestion
- Palette [interface customization](../enterprise-version/system-management/customize-interface.md) has been expanded with the ability to hide the login panel image and Spectro Cloud sidebar logo, change the default documentation domain, and change the **Sign in to your tenant** logo.
```

- The management of
  [cluster profile variables](../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables/define-profile-variables.md)

>   has been improved in the Palette UI. Users can now review and mcodify variable values during

Reviewer 2: ```suggestion
  has been improved in the Palette UI. Users can now review and modify variable values during
```

  [cluster profile version changes](../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md).

> - Palette uses Velero version 1.15 internally. Existing clusters with configured backups will be automatically updated
>   to Velero version 1.15, ensuring continuous access to Palette backup and restore functionality. Refer to the
>   [Backup and Restore](../clusters/cluster-management/backup-restore/backup-restore.md) page to learn more about backup
>   and restore tools in Palette.

Reviewer 2: ```suggestion
- Velero has been upgraded to version 1.15, which is used internally by Palette for backing up and restoring clusters. Existing clusters with backups configured will be automatically updated to Velero version 1.15, ensuring continuous access to backup and restore functionality. Refer to the
  [Backup and Restore](../clusters/cluster-management/backup-restore/backup-restore.md) page to learn more about backup
  and restore tools in Palette.
```

- Palette now uses Cluster API Provider AWS (CAPA) version 2.7.1 internally. Refer to the
  [documentation](https://github.com/kubernetes-sigs/cluster-api-provider-aws/tree/v2.7.1) for further information.
- [Self-hosted Palette](../enterprise-version/enterprise-version.md) now supports anonymous SMTP mode, allowing users to

>   authenticate with a username and password. We recommend using authenticated SMTP wherever possible. Refer to the

Reviewer 2: ```suggestion
  authenticate without a username and password. We recommend using authenticated SMTP wherever possible. Refer to the
```

  [Configure SMTP](../enterprise-version/system-management/smtp.md) guide for further information.

### Edge

#### Features

> - <TpBadge /> The new Appliance Studio is a lightweight Graphic User Interface (GUI) application allows allows you to

Reviewer 2: ```suggestion
- <TpBadge /> The new Appliance Studio is a lightweight Graphical User Interface (GUI) application allows you to
```

  build, save, edit, and manage the two configuration files that are essential to the EdgeForge process, with zero risk
  of syntax errors.

  Refer to the Prepare User Data and Argument Files guide to learn more.

- <TpBadge /> The Palette Optimized Canonical Kubernetes distribution is now an available selection for the [EdgeForge
  workflow](../clusters/edge/edgeforge-workflow/edgeforge-workflow.md) of preparing an Edge host with all the required
  components and dependencies. Refer to the [Build Edge
  Artifacts](../clusters/edge/edgeforge-workflow/palette-canvos/palette-canvos.md) for further information.

#### Improvements

- Improved the upgrade process for the Palette agent and increased its reliability.

> - Palette now supports upload one or more content bundles as long as your Edge host has enough physical storage and you

Reviewer 2: ```suggestion
- Palette now supports uploading one or more content bundles as long as your Edge host has enough physical storage and you
```

  have allocated sufficient storage to your registry. Refer to the
  [Upload Content Bundle](../clusters/edge/local-ui/cluster-management/upload-content-bundle.md) guide for further
  information.
- [Local UI](../clusters/edge/local-ui/local-ui.md) has now exited Tech Preview and is ready to use for production
  workloads. Check out the [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) guide for
  further details.
- A new troubleshooting script is now available to allow you to debug common errors such as Edge hosts failing to
  register, pod failures, or provisioning errors. Refer to the Collect Support Bundles for Edge Cluster Troubleshooting
  reference page for further details.

#### Deprecations and Removals

- The `stylus.installationMode`
  [Edge Installer Configuration](../clusters/edge/edge-configuration/installer-reference.md) flag is deprecated. We

>   recommend using the `stylus.managementMode` flag instead, which has two allowed values: `central` means the Edge host
>   is connected to Palette, `local` means the Edge host has no connection to a Palette instance. Refer to the

Reviewer 2: ```suggestion
  recommend using the `stylus.managementMode` flag instead, which has two allowed values: `central`, which means the Edge host
  is connected to Palette, and `local`, which means the Edge host has no connection to a Palette instance. Refer to the
```

#### Features

- The `content` command of the [Palette CLI](../automation/palette-cli/palette-cli.md) now has an `upload` subcommand.
  This subcommand allows you to build a content bundle and upload it to a locally managed Edge host through Local UI.
  Once the upload is complete, you can provision clusters locally using the uploaded content when the host does not have
  a connection to a central Palette instance or an image repository.

  Refer to the Upload Content Bundle reference page for further information.

- The `spectrocloud_appliance` resource now supports [remote shell](../clusters/edge/cluster-management/remote-shell.md)

>   activation, allowing you to troubleshoot edge hosts by initiating an SSH connection from Palette. For more

Reviewer 2: ```suggestion
  activation, allowing you to troubleshoot Edge hosts by initiating an SSH connection from Palette. For more
```

  information, refer to the Spectro Cloud Terraform provider
  [documentation](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs).

#### Improvements

#### Deprecations and Removals

- The `tc` subcommand of the [Palette CLI](../automation/palette-cli/palette-cli.md) is deprecated. This command

>   provided functionality for deploying target clusters using the Palette CLI. We recommend to use the

Reviewer 2: ```suggestion
  provided functionality for deploying target clusters using the Palette CLI. We recommend using the
```

  [Spectro Cloud Terraform provider](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) for
  cluster deployment automation.

> - The `spectrocloud_macro` Terraform resource is deprecated. We recommend to use the `spectrocloud_macros` resource to

Reviewer 2: ```suggestion
- The `spectrocloud_macro` Terraform resource is deprecated. We recommend using the `spectrocloud_macros` resource to
```

<!-- prettier-ignore-start -->

- The <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" /> versions 1.31.x and 1.32.x packs now support cluster deployments to AWS.

> - The <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" /> version 1.32.x packs now support cluster deployments to AKS.

Reviewer 2: ```suggestion
- The <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" /> version 1.32.x pack now supports cluster deployments to AKS.
```
